install.ps1

```diff
@@ -0,0 +1,52 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
+[Security.Principal.WindowsBuiltInRole] 'Administrator'))
+{
+	Write-Warning 'You do not have Administrator rights to run this script! Please re-run this script as an Administrator!'
+	Break
+} else {
+
+	Write-Host "Downloading VirtualBox, VirtualBox Extensions, and Vagrant"
+
+	If (Test-Path "C:\Program Files\Oracle\VirtualBox\") {
```

Comment:
This needs to be updated to use `%programfiles%` instead of `C:\Program Files`
http://stackoverflow.com/questions/9594066/how-to-get-program-files-x86-env-variable

Additionally I may have my VirtualBox installed in any drive outside of Program files.
For example in my case this is:
`C:\DevInstall\VirtualBox`